tests/acceptance/FormEditorCreateCustomFieldCest.php

```diff
@@ -6,19 +6,32 @@
 use MailPoet\Test\DataFactories\Segment;
 
 class FormEditorCreateCustomFieldCest {
-  public function createCustomSelect(\AcceptanceTester $i) {
+  private function prepareTheForm(\AcceptanceTester $i) {
```

Comment:
I can see that this function and also editTheForm is used at the beginning of every test in this suite so I guess you can just put the code into _before hook function that is triggered before every test.

Reply:
Yes, will make such a method.